scripts/merge-buddyup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# syntax:
+# merge-buddyup.sh locale-dir/
+
+function usage() {
+    echo "syntax:"
+    echo "merge-buddyup.sh locale-dir/"
+    exit 1
+}
+
+# check if file and dir are there
+if [[ ($# -ne 1) || (! -d "$1") ]]; then
+    usage
+fi
+
+POTFILE=`find $1 -type f -name "buddyup.pot"`
```

Comment:
Nit: using backticks for subshells is lame, mainly because they don't nest, I can't easily type them in code blocks, and because they are all slanty. `$(find ...)` is generally considered better style.